src/app/researcher/study/request/[memberIdentifier]/study-proposal.tsx

```diff
@@ -1,18 +1,19 @@
 'use client'
 import React from 'react'
 import { useForm } from '@mantine/form'
-import { Anchor, Button, Divider, FileInput, Flex, Group, Paper, Stack, Text, TextInput } from '@mantine/core'
+import { TextInput } from '@/components/form'
+import { Anchor, Button, Divider, FileInput, Flex, Group, Paper, Stack, Text } from '@mantine/core'
 import { useRouter } from 'next/navigation'
 import { useMutation } from '@tanstack/react-query'
 import { onCreateStudyAction } from './actions'
-import { StudyProposalFormValues, studyProposalSchema, zodResolver } from './studyProposalSchema'
+import { StudyProposalFormValues, studyProposalSchema, zodResolver } from './schema'
 
 export const StudyProposalForm: React.FC<{ memberId: string; memberIdentifier: string }> = ({ memberId }) => {
     const router = useRouter()
 
-    const handleFileChange = (key: string, file: File | null) => {
-        studyProposalForm.setFieldValue(key as keyof StudyProposalFormValues, file)
-    }
+    // const handleFileChange = (key: string, file: File | null) => {
```

Comment:
still needed?

Reply:
heh, I'm not sure what this was intended do do, I commented it out to see if anything broke.  since it didn't we should remove it.  ty!